apps/sr-frontend/src/app/Header.tsx

```diff
@@ -148,7 +148,7 @@ function HeaderContent({ anchor }: { anchor: string }) {
   const theme = useTheme()
   const isXL = useMediaQuery(theme.breakpoints.up('xl'))
   const isMobile = useMediaQuery(theme.breakpoints.down('md'))
-  const { t } = useTranslation('header')
+  const { t } = useTranslation('ui')
```

Comment:
I think I would prefer to have the `header` be specifically for the header here. I think `ui` might become a shared translation namespace for translations that are done for both SRO and GO.